scripts/unit.sh

```diff
@@ -8,6 +8,9 @@ source ${OKTA_HOME}/${REPO}/scripts/setup.sh
 export TEST_SUITE_TYPE="jsunit"
 export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/unit"
 
+# build is required to run E2E tests
+yarn build
```

Comment:
Do want to fail on any errors here?

Reply:
Thats what the `-e` flag to bash should do https://github.com/okta/okta-auth-js/pull/450/files/a0cfb967c6ec2978844601f54ccde51872773ae8#diff-e26b1f87e6f1791f5f3a9d9069fb998dR1